From a5c23986add86dc1620f6e1da94a0cecf5865e4a Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 12 Mar 2018 16:47:00 +0800
Subject: [PATCH] media: soc_camera/rockchip: Fix compile warning

aptina/imx/ov_camera_module.c:
Fix this warning: this if clause does not guard... [-Wmisleading-indentation]

Change-Id: I788d4d4d04dd2b0b7c41e1a041e9084c62b1975c
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/media/i2c/soc_camera/rockchip/aptina_camera_module.c | 3 ++-
 drivers/media/i2c/soc_camera/rockchip/imx_camera_module.c    | 3 ++-
 drivers/media/i2c/soc_camera/rockchip/ov_camera_module.c     | 3 ++-
 3 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/drivers/media/i2c/soc_camera/rockchip/aptina_camera_module.c b/drivers/media/i2c/soc_camera/rockchip/aptina_camera_module.c
index 9782de363a8b..dd20bdf5d4c5 100644
--- a/drivers/media/i2c/soc_camera/rockchip/aptina_camera_module.c
+++ b/drivers/media/i2c/soc_camera/rockchip/aptina_camera_module.c
@@ -603,10 +603,11 @@ int aptina_camera_module_s_stream(struct v4l2_subdev *sd, int enable)
 		if (!IS_ERR_OR_NULL(cam_mod->custom.set_flip))
 			cam_mod->custom.set_flip(cam_mod);
 
-		if (cam_mod->update_config)
+		if (cam_mod->update_config) {
 			ret = aptina_camera_module_write_config(cam_mod);
 			if (IS_ERR_VALUE(ret))
 				goto err;
+		}
 
 		ret = cam_mod->custom.start_streaming(cam_mod);
 		if (IS_ERR_VALUE(ret))
diff --git a/drivers/media/i2c/soc_camera/rockchip/imx_camera_module.c b/drivers/media/i2c/soc_camera/rockchip/imx_camera_module.c
index 34317d5fd30a..ad0ba0f1aa17 100644
--- a/drivers/media/i2c/soc_camera/rockchip/imx_camera_module.c
+++ b/drivers/media/i2c/soc_camera/rockchip/imx_camera_module.c
@@ -398,10 +398,11 @@ int imx_camera_module_s_stream(struct v4l2_subdev *sd, int enable)
 			ret = -EINVAL;
 			goto err;
 		}
-		if (cam_mod->update_config)
+		if (cam_mod->update_config) {
 			ret = imx_camera_module_write_config(cam_mod);
 			if (IS_ERR_VALUE(ret))
 				goto err;
+		}
 
 		ret = cam_mod->custom.start_streaming(cam_mod);
 		if (IS_ERR_VALUE(ret))
diff --git a/drivers/media/i2c/soc_camera/rockchip/ov_camera_module.c b/drivers/media/i2c/soc_camera/rockchip/ov_camera_module.c
index 2495de3e8adb..9e8110e5636c 100644
--- a/drivers/media/i2c/soc_camera/rockchip/ov_camera_module.c
+++ b/drivers/media/i2c/soc_camera/rockchip/ov_camera_module.c
@@ -418,10 +418,11 @@ int ov_camera_module_s_stream(struct v4l2_subdev *sd, int enable)
 			ret = -EINVAL;
 			goto err;
 		}
-		if (cam_mod->update_config)
+		if (cam_mod->update_config) {
 			ret = ov_camera_module_write_config(cam_mod);
 			if (IS_ERR_VALUE(ret))
 				goto err;
+		}
 
 		ret = cam_mod->custom.start_streaming(cam_mod);
 		if (IS_ERR_VALUE(ret))
-- 
2.35.3

